ntbs-service/Startup.cs

```diff
@@ -205,17 +208,25 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 
             app.UseMvc();
 
-            app.UseHangfireDashboard("/hangfire", new DashboardOptions
-            {
-                Authorization = new[] { new HangfireAuthorisationFilter(GetAdminRoleName()) }
-            });
-            app.UseHangfireServer(new BackgroundJobServerOptions { WorkerCount = 1 });
+            ConfigureHangfire(app);
```

Comment:
ah, I thought you have already wrapped this in an if statement

Reply:
i have in the merge - good spot! :)